### PR TITLE
Classes to read DCS Data Points 

### PR DESCRIPTION
**Pull request by chiarazampolli**
The classes are taken from ADAPRO, with small modifications, in order to not have to include the full ADAPRO in O2. 
The DCS data points are to be used in calibration.

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Not sure the error "Check PR with clang-format / build (pull_request_target) " is related to my PR. I ran clang-format, and in the log I could not see anything special. Maybe I am just blind. @ktf , @shahor02 , any idea? Maybe your eyes are better.

**Comment by ktf** [comment]
Ignore the thing for now.

**Comment by shahor02** [comment]
@chiarazampolli : the DCS code does not comply with high O2 standards :), could you modernize it ? :
````
/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/build_o2checkcode_o2/0/Detectors/DCS/include/DetectorsDCS/DataPointIdentifier.h:174:3: warning: use '= default' to define a trivial destructor [modernize-use-equals-default]
  ~DataPointIdentifier() noexcept {}
  ^                               ~~
                                  = default;
````

**Comment by chiarazampolli** [comment]
Sure, I just cut and pasted the content of the classes :-) Have you seen if this is the only case?
I will take care tonight, I am away today, and in addition, my phone (with which I am now connected) is 1/2 empty :-(

**Comment by chiarazampolli** [comment]
Ok, hopefully all is fine now, I saw that the error was from the o2checkcode.

File: Detectors/Calibration/CMakeLists.txt.mine [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is distributed
```
Comment (shahor02) [inline Detectors/Calibration/CMakeLists.txt.mine]: is this a stray file?

File: Detectors/DCS/src/DetectorsDCSLinkDef.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// In applying this license CERN does not waive the privileges and immunities
```
Comment (shahor02) [inline Detectors/DCS/src/DetectorsDCSLinkDef.h:7]: the licence is cut

File: dependencies/O2Dependencies.cmake [hunk 3]
```diff
@@ -152,4 +152,7 @@ find_package(O2GPU)
 
 find_package(FastJet)
 
+#find_package(ADAPRO)
```
Comment (shahor02) [inline dependencies/O2Dependencies.cmake]: we should probably remove this.

**Review by shahor02 — COMMENTED** [review]
Hi Chiara,
please see below.
Cheers,
 Ruben

File: dependencies/O2Dependencies.cmake (hunk 3 above)
Comment (chiarazampolli) [reply dependencies/O2Dependencies.cmake]: done, yes. It went in by mistake.

File: Detectors/DCS/src/DetectorsDCSLinkDef.h (hunk 2 above)
Comment (chiarazampolli) [reply Detectors/DCS/src/DetectorsDCSLinkDef.h:7]: Yes, sorry, I worked in a "funny" mode yesterday morning, and I did not notice.

File: Detectors/Calibration/CMakeLists.txt.mine (hunk 1 above)
Comment (chiarazampolli) [reply Detectors/Calibration/CMakeLists.txt.mine]: yes. removed.

**Review by shahor02 — APPROVED** [review]

**Pull request merged**